rpc/src/v1/helpers/errors.rs

```diff
@@ -271,6 +272,14 @@ pub fn from_transaction_error(error: EthcoreError) -> Error {
 	}
 }
 
+pub fn from_rlp_error(error: DecoderError) -> Error {
```

Comment:
why are so many of the functions in the `error` module suffixed with `error`? it makes it a bit awkward when calling (`error::from_rlp_error()` vs `error::from_rlp`) as well as being inconsistent with other functions in the module, e.g. `error::signer_disabled`